### ISSUE TITLE
Provide event details

### ISSUE THREAD

**Issue by Abdull** [fullcalendar/fullcalendar#5942]
### Checklist

  - [x] I've already searched through [existing tickets](https://fullcalendar.io/issues)
  - [x] Other people will find this feature useful

### Feature Description

Every calendar needs to have details for an event.

**Comment by acerix** [comment]
Do you mean something like a popup with more details when an event is clicked?  You can include more details in the extendedProps and display it with `eventContent`.

**Comment by Abdull** [comment]
Yes, that's what I meant. Shouldn't this be provided out of the box? Any other calendar I can think of provides this, e.g. Google Calendar, Outlook Calendar.

**Comment by ganySA** [comment]
I dont think this is critical to be honest - its so implementation specific / framework specific. I have built a full calendar - google calendar 'clone' and implemented my own popup with the events that are available.

If you do decide to implement please make it optional....

![image](https://user-images.githubusercontent.com/3189206/100969463-c8044e00-353b-11eb-8510-58b4d71f6ffd.png)

**Comment by Abdull** [comment]
Then why is it called *full*calendar?

**Comment by dakala** [comment]
@ganySA That looks really neat, sir! What's the relationship between the mini-calendar on the left and the fullcalendar on the right? How was it achieved?

**Comment by dwasyl** [comment]
@ganySA Can I ask how you achieved the popover with interactive elements? I've built something not dissimilar, but the Popover elements (based on a Bootstrap Popover) aren't interactive (clickable).

**Comment by quirkyexplorer** [comment]
@ganySA Hi Im wanting to built something similar, can you provide example implementation the feature you presented?

**Comment by alifdarsim** [comment]
For those who like me wasting time to create a popup like GoogleCalendar and end up with buggy popups. There is another library with native popup function similar to GoogleCalendar https://github.com/nhn/tui.calendar
